agent/engine/docker_task_engine.go

```diff
@@ -474,7 +474,10 @@ func (engine *DockerTaskEngine) GetTaskByArn(arn string) (*api.Task, bool) {
 
 func (engine *DockerTaskEngine) pullContainer(task *api.Task, container *api.Container) DockerContainerMetadata {
 	pullStart := time.Now()
-	defer seelog.Infof("Finished pulling container %v. Lock acquisition and pull took %v.", container, time.Since(pullStart).String())
+	defer func(startTime time.Time) {
+		seelog.Infof("Finished pulling container %v. Lock acquisition and pull took %v.", container, time.Since(startTime).String())
```

Comment:
nit: The second string format can `%s` to `Infof`, instead of `%v`. Also, I'm wondering if the phrase `Lock acquisition` should be logged. Can it just be `Finished pulling container %v in %s`?